src/lib/maxim.ts

```
@@ -521,6 +531,8 @@ export class Maxim {
 					nodes: promptChainVersionAndRules.fallbackVersion!.config
 						? promptChainVersionAndRules.fallbackVersion!.config.nodes.filter((n) => "prompt" in n)
 						: [],
+					run: (input: string, options?: { variables?: { [key: string]: string } }) =>
+						this.APIService.promptChain.runPromptChainVersion(promptChainVersionAndRules.fallbackVersion!.id, input, options),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)